### ISSUE TITLE
Exporting hashtable to new (empty) file doesn't transfer as expected

### ISSUE THREAD

**Issue by SamScripting** [dfinke/ImportExcel#1551]
![image](https://github.com/dfinke/ImportExcel/assets/155124893/0a782895-23f4-4b20-8149-a946db44d1da)

I assume it's because when adding a hashtable, it's expecting to match based on table headers, but since none exist, it's doing (whatever the technical reason for this is). If I want to export a hash table to a fresh excel sheet, what do I need to do? I saw the New-ExcelTable which I understand, but I'm not seeing an adjacently paired way to create headers on that table, pipe in an array to fill a range to create headers, or something along those lines so I can then attempt to pipe in a hashtable to populate the data.

Arrays populate down the column just fine, hence me suspecting it's expecting to find a header association.

**Comment by scriptingstudio** [comment]
```powershell
[pscustomobject]$assetdata | export-excel ...
```

**Comment by ninmonkey** [comment]
The problem is you're piping a hashtable, instead of a list of hashtables. So it's enumerating on the properties of the hash itself
try this
```ps1
$data = @( 
    [pscustomobject]@{ Name = 'bob' } 
    [pscustomobject]@{ Name = 'jen' } 
) | Export-Excel -Show -TableName 'employees' -TableStyle Light2
```

**Comment by SamScripting** [comment]
Yep, both work. Learning new things sure makes a man feel stupid sometimes.

**Comment by dfinke** [comment]
You're learning how PowerShell works. Takes time. Good news, this will help you in all your PowerShell efforts.
